packit.yaml

```diff
@@ -5,4 +5,6 @@ upstream_project_name: cockpit-ostree
 downstream_package_name: cockpit-ostree
 actions:
   post-upstream-clone: make cockpit-ostree.spec
-  create-archive: make dist-gzip
+  create-archive:
+    - make dist-gzip
+    - make dist-gzip-dump
```

Comment:
OOI, what was wrong with letting dist-gzip output the path by itself?

Reply:
Nothing really, I am still trying to figure out what is the problem. Locally it worked just fine without any additional target. Here it does not work. It outputs only a few lines from `make dist-gzip` so I thought maybe it does not see the last line, so I tried if having separate target would help. It did not, even though it is now visible in the output. Still playing around. Once I find out the solution I'll clean up after myself.